### DISCUSSION TITLE
[Feature] Smooth animations and gestures for gallery

### DISCUSSION THREAD

**Discussion by hyperstown** [immich-app/immich#10048] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently in web app opening and browsing through images looks very basic and on mobile (technically desktop too) it doesn't support any gestures. This could be improved with quite popular (23k+ stars on GitHub) JS library [Photoswipe](https://photoswipe.com/). 
While I understand gestures are somewhat low priority since we have mobile app it still would be great if web app would support those. 
As for image animations I'd say those are a little bit more needed. Currently web app doesn't seem smooth. Even though everything works quite fast, lack of any smooth transitions makes everything feel laggy. 
I really recommend checking out Photoswipe demo to see just how much of a difference transitions make.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Thanks, I appreciate the suggestion! Will check it out

**Comment by ashulgach** [comment]
An additional perspective - this can help people that are sensitive to flashing lights. As it is, on the webapp when viewing a fullsize photo and pressing right arrow to scroll, the image cuts to black and fades in extremely quickly, creating a strobe-like effect if bright photos are viewed. Animating the scroll or not cutting to black would make a big difference for accessibility.
